src/drivers/build/drivers-384.111.dockerfile

```diff
@@ -57,13 +86,39 @@ RUN apt-get -y update && \
 
 WORKDIR $STAGE_DIR
 
-ENV NVIDIA_VERSION=384.111
+ENV NVIDIA_VERSION=384.111 \
+    OFED_VERSION=4.2-1.2.0.0 \
+    OS_VERSION=ubuntu16.04 \
+    ARCHITECTURE=x86_64
+
+ENV MLNX_OFED_STRING=MLNX_OFED_LINUX-${OFED_VERSION}-${OS_VERSION}-${ARCHITECTURE}
 
 RUN wget --no-verbose http://us.download.nvidia.com/XFree86/Linux-x86_64/$NVIDIA_VERSION/NVIDIA-Linux-x86_64-$NVIDIA_VERSION.run && \
     chmod 750 ./NVIDIA-Linux-x86_64-$NVIDIA_VERSION.run && \
     ./NVIDIA-Linux-x86_64-$NVIDIA_VERSION.run --extract-only && \
     rm ./NVIDIA-Linux-x86_64-$NVIDIA_VERSION.run
 
+RUN echo "wget -q -O - http://www.mellanox.com/downloads/ofed/MLNX_OFED-$OFED_VERSION/$MLNX_OFED_STRING.tgz | tar xzf -" && \
+    wget -q -O - http://www.mellanox.com/downloads/ofed/MLNX_OFED-$OFED_VERSION/$MLNX_OFED_STRING.tgz | tar xzf - && \
```

Comment:
maybe untar the file in installation script? the drivers image is already 3G large.

Reply:
@xudifsd  We should consider some inter-networking environment may can't access to outer-networking. So we have to prepare all downloading and installation dependency in the docker image.

Reply:
yes, I know, so I only ask not to untar.

Reply:
Got it, I will try to optimize it.